examples/main/views/custom-interaction.js

```diff
@@ -62,7 +64,7 @@ const propTypes = {
   height: PropTypes.number.isRequired
 };
 
-export default class TiltExample extends Component {
+export default class Example extends Component {
```

Comment:
this is copy & pasted from the `custom-interactions` example - so no idea why that was named `custom-interactions`. Happy to switch this to "MainExample" or "PerspectiveExample" or something like that though.